scripts/test.sh

```diff
@@ -17,7 +17,7 @@ export RUST_BACKTRACE=1
 
 if [ -n "$PACKAGE" ];
 then
-  cargo test -p $PACKAGE --all-features --no-fail-fast
+  cargo test -p $PACKAGE --all-features --no-fail-fast $TEST
```

Comment:
Can run a single test with this without manually editing the script each time.

Reply:
Nice.